src/utils/sort.js

```diff
@@ -0,0 +1,9 @@
+import {SortType} from '../constants.js';
+
+export const sort = {
+  [SortType.DAY]: (points) => points.sort(),
+  [SortType.EVENT]: (points) => points.sort(),
```

Comment:
Здесь лучше сделать возврат поинтов без изменения
```suggestion
  [SortType.EVENT]: (points) => points,
```